ymer-test-junit4/src/main/java/com/avanza/ymer/YmerConverterTestBase.java

```diff
@@ -121,6 +121,20 @@ private MirroredObjectTestHelper getMirroredObjectHelper(Class<?> objectClass) {
 
 	protected abstract MongoConverter createMongoConverter(MongoDbFactory mongoDbFactory);
 
+	private MongoConverter createMongoConverter() {
+		MongoConverter converter = createMongoConverter(dummyMongoDbFactory);
+		if (converter instanceof AbstractMongoConverter) {
+			// In a real app, the instance of MongoConverter is usually
+			// registered as a spring bean, which will make "afterPropertiesSet"
+			// be called automatically. But here, we are not in a spring context
+			// and therefore will need to call it manually since
+			// MappingMongoConverter only updates its internal state of
+			// "conversionService" once this is called.
+			((AbstractMongoConverter) converter).afterPropertiesSet();
+		}
+		return converter;
+	}
```

Comment:
Bra! Jag tror att detta behövs även i `ymer-test-junit5`

Reply:
Sure, have added this now in a separate commit.